low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -191,6 +191,8 @@ SERVICEMONITOR:
       - salt-minion.service
       - glusterd.service
       - scsi-network-relay.service
+      - kafka.service
```

Comment:
so when systemd interfaces not in places, sspl wont get any info even whether its enabled / disabled & will be ignored ? 
@Sumedh-a-k

Reply:
yes, since it won't find any `Unit` for kafka.service, it will be removed from the list of services to monitor..

Reply:
then we are good